internal/store/deployment_test.go

```diff
@@ -169,6 +174,7 @@ func TestDeploymentStore(t *testing.T) {
         kube_deployment_status_replicas_unavailable{deployment="depl2",namespace="ns2"} 0
         kube_deployment_status_replicas_updated{deployment="depl2",namespace="ns2"} 1
         kube_deployment_status_replicas{deployment="depl2",namespace="ns2"} 10
+        kube_deployment_status_replicas_ready{deployment="depl2",namespace="ns2"} 5
```

Comment:
Is ready always the same as kube_deployment_status_replicas_available? Judging by these tests at least? 🤔

Reply:
@lilic 
Available and Ready replicas count will not necessarily be same
Available replicas are more like a subset of ready replicas (Replicas which are ready for a certain amount of time configurable based on spec.minReadySeconds)